src/api/Services/FileSystemOperations.cs

```diff
@@ -79,8 +79,8 @@ public async Task<Result> CreateFileSystem(string fileSystemName, string owner,
 			}
 			catch (Exception ex)
 			{
-				string Message = $"Error while trying to query for the existence of container '{fileSystemName}' in account '{dlsClient.AccountName}': '{ex.Message}'.";
-				log.LogError(ex, Message);
+				string Message = "Error while trying to query for the existence of container '{fileSystemName}' in account '{dlsClientAccountName}': '{exMessage}'.";
+				log.LogError(ex, Message, fileSystemName, dlsClient.AccountName, ex.Message);
 				result.Success = false;
 				result.Message = Message;

```

Comment:
The message that is returned to the client won't have interpolated strings here.